libpod/info.go

```diff
@@ -0,0 +1,119 @@
+package libpod
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/docker/docker/pkg/system"
+)
+
+// InfoData holds the info type, i.e store, host etc and the data for each type
+type InfoData struct {
+	Type string
+	Data map[string]interface{}
+}
+
+// top-level "host" info
+func (r *Runtime) hostInfo() map[string]interface{} {
```

Comment:
Can this return an error as well instead of using infoErr?

Reply:
fixed